pandas/io/json/table_schema.py

```diff
@@ -308,7 +308,7 @@ def parse_table_schema(json, precise_float):
 
     for col, dtype in dtypes.items():
         if dtype == 'timedelta64[ns]':
-            df[col] = df[col].apply(Timedelta)
+            df[col] = to_timedelta(df[col])
 
     df = df.astype(dtypes)
```

Comment:
Is there any conflict between this and the loop above? Wondering if we should be removing any timedelta calls from the `dtypes` dict since they get accounted for in the loop